### PR TITLE
Fixes Sanity test errors

### LINKED ISSUE #62: Saniry tests are failing
## Summary
When running sanity tests on a pull request, the following errors are seen

ERROR: Found 7 shebang issue(s) which need to be resolved:
See documentation for help: https://docs.ansible.com/ansible/devel/dev_guide/testing/sanity/shebang.html
ERROR: Jenkinsfile:1:1: unexpected non-module shebang: b'#!/usr/bin/env groovy'
ERROR: ci/build_release:1:1: unexpected non-module shebang: b'#!/bin/bash'
ERROR: ci/parse-changelog.sh:1:1: unexpected non-module shebang: b'#!/bin/bash -ex'
ERROR: ci/publish_to_galaxy:1:1: unexpected non-module shebang: b'#!/bin/bash'
ERROR: ci/test.sh:1:1: unexpected non-module shebang: b'#!/bin/bash -x'
ERROR: roles/conjur_host_identity/tests/test.sh:1:1: unexpected non-module shebang: b'#!/bin/bash -e'
ERROR: tests/conjur_variable/test.sh:1:1: unexpected non-module shebang: b'#!/bin/bash -e'

ERROR: Found 3 yamllint issue(s) which need to be resolved:
See documentation for help: https://docs.ansible.com/ansible/devel/dev_guide/testing/sanity/yamllint.html
ERROR: roles/conjur_host_identity/tests/policy/root.yml:2:3: unparsable-with-libyaml: None - could not determine a constructor for the tag '!policy'
ERROR: secrets.yml:2:17: unparsable-with-libyaml: None - could not determine a constructor for the tag '!var'
ERROR: tests/conjur_variable/policy/root.yml:2:3: unparsable-with-libyaml: None - could not determine a constructor for the tag '!policy'
ERROR: The 2 sanity test(s) listed below (out of 32) failed. See error output above for details.

## Steps to Reproduce

On a pull request

## Expected Results

Sanity tests should pass

## Actual Results (including error logs, if applicable)
Sanity fails

## Reproducible
   * [x] Always 
   * [ ] Sometimes
   * [ ] Non-Reproducible
   
## Version/Tag number
What version of the product are you running? Any version info that you can share is helpful. 
For example, you might give the version from Docker logs, the Docker tag, a specific download URL, 
the output of the `/info` route, etc.

## Environment setup
Can you describe the environment in which this product is running? Is it running on a VM / in a container / in a cloud? 
Which cloud provider? Which container orchestrator (including version)? 
The more info you can share about your runtime environment, the better we may be able to reproduce the issue.

### PR DESCRIPTION
**Pull request by rpothier**
### What does this PR do?
This fixes the sanity failures buy adding ignore-2.12.txt
There were failures on devel, which needs the 2.12 ignore file.

( Also removes a space from the README.md)

### What ticket does this PR close?
Resolves #62

#### Change log
- [ ] The CHANGELOG has been updated, or
- [x] This PR does not include user-facing changes and doesn't require a CHANGELOG update

#### Test coverage
- [ ] This PR includes new unit and integration tests to go with the code changes, or
- [x] The changes in this PR do not require tests

#### Documentation
- [ ] Docs (e.g. `README`s) were updated in this PR, and/or there is a follow-on issue to update docs, or
- [x] This PR does not require updating any documentation

### CODE REVIEW

File: tests/sanity/ignore-2.12.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Jenkinsfile shebang
+tests/conjur_variable/test.sh shebang
+tests/conjur_variable/policy/root.yml yamllint:unparsable-with-libyaml
+roles/conjur_host_identity/tests/test.sh shebang
+roles/conjur_host_identity/tests/policy/root.yml yamllint:unparsable-with-libyaml # File loaded by summon utility (in Jenkinsfile), not via Python
```
Comment (izgeri) [inline tests/sanity/ignore-2.12.txt:5]: I think the comments for this line and line 10 are swapped. This is a policy file; secrets.yaml is a Summon fiile

Comment (izgeri) [reply tests/sanity/ignore-2.12.txt:5]: do we want a comment like this on line 3 too?

Comment (rpothier) [reply tests/sanity/ignore-2.12.txt:5]: good catch, swapping comments.

**Review by izgeri — APPROVED** [review]
LGTM

**Pull request merged**